QUEST_LV_0200.tsv

```diff
@@ -4989,13 +4989,13 @@ QUEST_LV_0200_20150323_004988	Search through Orville's friend
 QUEST_LV_0200_20150323_004989	Orville told you that he lost his friend while being chased by the monsters. Find Orville's friend.
 QUEST_LV_0200_20150323_004990	Orville's friend is dead. You should pass this sad story to Orville.
 QUEST_LV_0200_20150323_004991	{memo X}Go meet Tadas
-QUEST_LV_0200_20150323_004992	Someone needs your help at the land of Areluno Baron. Go meet him.
+QUEST_LV_0200_20150323_004992	Someone needs your help at the land of Baron Allerno. Go meet him.
 QUEST_LV_0200_20150323_004993	Get some tough string
-QUEST_LV_0200_20150323_004994	{memo X}Tadas asked you to get some strings since there arent' many strings to tie the sacks of grains. Hunt the monsters to some strings.
-QUEST_LV_0200_20150323_004995	{memo X}Take the tough strings to him
-QUEST_LV_0200_20150323_004996	{memo X}You've obtained enough strings. Hand them over to Taas.
+QUEST_LV_0200_20150323_004994	{memo X}Tadas asked you to get some rope since there aren't' many rope to tie the sacks of grains. Hunt the monsters to some rope.
```

Comment:
{memo X}Tadas asked you to get some rope since there aren't enough ropes to tie the sacks of grains. Hunt the monsters to obtain some rope.

There was an extra apostrophe at the end of "aren't", the second "rope" should be "ropes" and the second sentence lacked a verb.